### ISSUE TITLE
TLS Error should fix itself

### ISSUE THREAD

**Issue by ser** [HenriWahl/Nagstamon#795]
There are sometimes network issues between Nagstamon and my icinga2 server. 

![obraz](https://user-images.githubusercontent.com/231537/149246098-29548f93-83f0-4318-8b5d-ae51b462cfd0.png)

Clicking "fix problem" fixes it indeed, but is it really required to click a button? In my opinion Nagstamon should try to fix it itself in some time intervals.

**Comment by hamster65** [comment]
I have to click this at least every morning and after my laptop wakes up from standby. I think it started after I upgraded Check_mk. I use a self signed certificate and configured nagstatmon to ignore it.

**Comment by HenriWahl** [comment]
I see - I have the same problem. Looks like an timed out session.

**Comment by HenriWahl** [comment]
@ser which monitoring server do you use?

**Comment by HenriWahl** [comment]
@hamster65 do you see this `SSL/TLS problem`or `Authentication problem` every morning?

**Comment by ser** [comment]
I use my own nagstamon `icinga2api` connector - should I try to fix it in the connector only? I was sure problem is more general.

**Comment by hamster65** [comment]
I see it after I connect to the company network via VPN. The VPN connection goes up automatically after login or wakeup from standby. I can check tomorrow how it behaves in the office.

**Comment by HenriWahl** [comment]
@hamster which the error you see there exactly: `SSL/TLS problem` or `Authentication problem`?

**Comment by HenriWahl** [comment]
@ser - well should be somethng general.

**Comment by HenriWahl** [comment]
@ser @hamster65 please check with latest testing release 20220419.

**Comment by ser** [comment]
Installed, I'll report back if I see the error, thanks <3

**Comment by hamster65** [comment]
Seems to be fixed for me with the latest testing release. Thanks!

**Comment by ser** [comment]
I have just experienced error in 3.9-20220503

**Comment by ser** [comment]
in 3.9-20220606 also problem repeats

**Comment by ser** [comment]
problem persists in 3.9-20220808

**Comment by hamster65** [comment]
I still use 3.9-20220704, no issues there.

**Comment by ser** [comment]
maybe your network quality has improved, mine not...

**Comment by HenriWahl** [comment]
@ser Which Icinga2 Server do you use exactly? Icinga2API or IcingaWeb2?

**Comment by ser** [comment]
Indeed. But I am not an author of the library myself - do you suspect it's the culprit?

**Comment by HenriWahl** [comment]
@ser no, I found the problem-indicating-flag **tls_error** could easily be reset as it will be set again if there is really a problem. Can you ty the latest 3.9-20220815 version which contains this attempt?
